config/values/finance.yml

```diff
@@ -1,19 +1,19 @@
 finance:
-  adultdependantsgrant:
+   adultdependantsgrant:
```

Comment:
not sure why we are adding a space in front of this one?

Reply:
Think that was me copying and pasting, will amend